src/core.js

```diff
@@ -156,6 +156,10 @@ export function updateValue(el, vnode, { emit = true, force = false } = {}, even
   currentValue = currentValue || ''
 
   if (force || oldValue !== currentValue) {
+    if (['deleteByCut', 'deleteContent', 'deleteContentBackward', 'deleteContentForward'].includes(event?.inputType)) {
+      config = { ...config, short: true }
+    }
```

Comment:
What is this doing? Maybe add a comment on why this is needed?

- [ ] Consider adding a comment

Reply:
this is also from a previous commit `b3655e2` which was merged to master but beta branch is behind. looked into it and found that this is related to the masker when we are adding the `accumulator` chars. Setting short to true means that when you are deleting the char can be removed as well. Maybe we can have the same comment here as from the masker.js `short` param comment: `to keep the string as short as possible (not append extra chars at the end)`
@RonaldJerez please feel free to correct/clarify if I am missing something

Reply:
the logic automatically adds static characters to the input as you type.  But during a delete operation we do not want that to happen otherwise it will not allow you to delete as it will just put the static characters back.  adding 'short' removes the profiling feature.